eng/performance/maui_scenarios_android.proj

```diff
@@ -11,6 +11,10 @@
     <PreparePayloadOutDirectoryName>scenarios_out</PreparePayloadOutDirectoryName>
     <PreparePayloadWorkItemBaseDirectory Condition="'$(TargetsWindows)' == 'true'">$(CorrelationPayloadDirectory)$(PreparePayloadOutDirectoryName)\</PreparePayloadWorkItemBaseDirectory>
     <PreparePayloadWorkItemBaseDirectory Condition="'$(TargetsWindows)' != 'true'">$(CorrelationPayloadDirectory)$(PreparePayloadOutDirectoryName)/</PreparePayloadWorkItemBaseDirectory>
+
+    <_MSBuildArgs Condition="'$(RuntimeFlavor)' == 'mono'">/p:UseMonoRuntime=true</_MSBuildArgs>
+    <_MSBuildArgs Condition="'$(RuntimeFlavor)' == 'coreclr'">/p:UseMonoRuntime=false;/p:AndroidEnableMarshalMethods=false</_MSBuildArgs>
```

Comment:
```suggestion
    <_MSBuildArgs Condition="'$(RuntimeFlavor)' == 'coreclr' and '$(PublishAot)' != 'true'">/p:UseMonoRuntime=false</_MSBuildArgs>
```
`AndroidEnableMarshalMethods ` is not needed anymore as of: https://github.com/dotnet/android/pull/9912